README.md

```diff
@@ -45,8 +45,31 @@ For other systems, please check the above requirements and use your correspondin
 TBD
 
 ## Windows
-
-TBD
+### GL-Node Builder
+- get [glbsp executable](http://glbsp.sourceforge.net/download.php)
+- place it in the root of the goom directory
+
+### mingw-w64 for makefile execution and portmidi usage
+- install [mingw-w64](http://mingw-w64.org/doku.php/download/mingw-builds)
+- add *\<mingw64 dir\>/mingw64/bin* to your user- or system path
+- you now can use *mingw32-make*
+
+### Sound (TODO: this surely can be done more straight forward)
```

Comment:
I would rather write a new (WIP) `WINDOWS.md` file and link it here.
This will not look so messy on the start page.
When we have reduced and cleaned up the instructions (or provide a script for the steps),
we can move the Windows section back to the main file

Reply:
+1